src/containers/Header/Menu.jsx

```diff
@@ -87,11 +87,16 @@ class Menu extends Component {
 
   render() {
     let { routes } = this.props;
+    const { inNetwork } = this.props;
 
     if (MODE !== 'mainnet') {
       routes = removeRoutes(routes, 'tokens');
     }
 
+    if (!inNetwork) {
+      routes = removeRoutes(routes, 'explorer', 'network', 'tokens');
+    }
```

Comment:
these should probably be defined in a constant - IE a list of a routes, which say where (or where not) they are valid